.github/workflows/engine-pull-request.yml

```diff
@@ -32,7 +32,7 @@ jobs:
   engine-checks-optional:
     name: ⚙️ Checks (Optional)
     uses: ./.github/workflows/engine-checks-optional.yml
-    needs: [engine-changed-files]
+    needs: ${{ (contains(github.event.pull_request.labels.*.name, 'CI: Clean build required') && [engine-changed-files] || [engine-changed-files, engine-checks] }}
```

Comment:
- Sometimes it may be necessary to run even `engine-checks-optional` without any waiting
- one **can opt-in** such behavior
   - we design a dedicated label for that (right now using "CI: Clean build required" for testing purposes)
   - adding such a label reduces the `needs` value back to `[engine-changed-files]`
   - e.g. returning the old behavior
- with such a change (and label "CI: Clean build required" on), this is the initial set of engine jobs:

TBD